test_runner/regress/test_broken_timeline.py

```diff
@@ -166,13 +173,16 @@ def test_timeline_init_break_before_checkpoint_recreate(neon_env_builder: NeonEn
     timeline_id = TimelineId("1080243c1f76fe3c5147266663c9860b")
 
     # Introduce failpoint during timeline init (some intermediate files are on disk), before it's checkpointed.
-    pageserver_http.configure_failpoints(("before-checkpoint-new-timeline", "return"))
-    with pytest.raises(Exception, match="before-checkpoint-new-timeline"):
+    pageserver_http.configure_failpoints(("before-checkpoint-new-timeline", pause_or_return))
```

Comment:
This is not a pausable failpoint.

Reply:
Fixed in [02b7c27](https://github.com/neondatabase/neon/pull/6009/commits/02b7c2743ff0e093dfc7a1f008e64c18ebb0d6c4) to avoid executor stall and possible deadlock with some other task being stuck there, even if it is impossible here.